src/logic/map_objects/tribes/worker_program.cc

```diff
@@ -722,7 +730,8 @@ buildferry
 ^^^^^^^^^^
 .. function:: buildferry
 
-   Adds a new instance of this tribe's ferry to the map at the worker's current location. Example::
+   **DEPRECATED** Adds a new instance of this tribe's ferry to the map at the worker's current
+location. Example::
```

Comment:
Just replace the whole comment and usage example with a line "DEPRECATED, use `createbob=TRIBENAME_ferry`" instead

Reply:
Done